tasks/main.yml

```diff
@@ -50,10 +50,11 @@
     mode: 0644
   register: dropin
 
-- name: Reload systemd
-  systemd:
-    daemon_reload: true
+- debug:
```

Comment:
Instead of using a debug task, the `register: dropin` above can be removed and replaced with `notify: Systemd daemon-reload`